tracer/test/snapshots/DataStreamsMonitoringTests.HandlesFanIn.verified.txt

```diff
@@ -145,6 +145,13 @@
             topic:data-streams-fan-in-out-1,
             type:kafka_produce
           ]
+        },
+        {
+          Tags: [
+            partition:2,
+            topic:data-streams-fan-in-out-2,
+            type:kafka_produce
+          ]
```

Comment:
this comes from the fact that I changed the test to produce 3 messages (instead of 2 before) from 3 messages received.